sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1080,20 +1080,27 @@ they do in non-recording mode.
 
 ==== Event Limitations
 
-Events returned from queue submissions when a queue is in the recording state
-may only be used as parameters to `handler::depends_on()` or as dependent
-events for queue shortcuts like `queue::parallel_for()` for submissions which
-are being recorded to the same modifiable `command_graph`.
+For queue submissions that are being recorded to a modifiable `command_graph`,
+the only events that can be used as parameters to `handler::depends_on()`, or
+as dependent events for queue shortcuts like `queue::parallel_for()`, are events
+that have been returned from queue submissions recorded to the same modifiable
+`command_graph`.
+
+Other limitations on the events returned from a submission to a queue in the
+recording state are:
 
 - Calling `event::get_info<info::event::command_execution_status>()` or
-`event::get_profiling_info()` on an event returned from a queue submission
-recorded to a graph will throw synchronously with error code `invalid`.
+`event::get_profiling_info()` will throw synchronously with error code `invalid`.
+
+- A host-side wait on the event will throw synchronously with error
+code `invalid`.
+
+- the event cannot be used outside of the recording scope.
```

Comment:
We should say how this will fail.
```suggestion
- Using the event outside of the recording scope will throw synchronously with error code
`invalid`.
```